### DISCUSSION TITLE
Want to understand the code

### DISCUSSION THREAD

**Discussion by Animeshsoni97** [apache/skywalking#9618] [General]
Dear owners,
**kindly explain a bit about code flow** and what technologies have been used in the Skywalking code in order to understand the project flow.
Its my persional intrest to study other's code. I have knowledge on java and its framworks also angular js. 
Thanks for your support in advance.

**Comment by Animeshsoni97** [comment]
also **can we run the code from ecclipse** post check out ??

How to debug also??

**Comment by wu-sheng** [comment]
In the official doc, there is no explanation for codes. Sources are open to read, this is how OSS works.
Which languages are you reading?

**Comment by Superskyyy** [comment]
We encourage you to participate in the upcoming Hacktoberfest as we just joined in the event. Kindly share your preferences so we can come up with some tasks for you to contribute and understand the code. @Animeshsoni97
